README.md

```diff
@@ -59,7 +59,7 @@ The tenants database connection details are configured [via environment variable
 Instructions to build a WAR file
 ============
 1. Clone the repository or download and extract the archive file to your local directory.
-2. Run `./gradlew clean bootWar` to build a traditional WAR file which will be created at `build/libs` directory.
+2. Run `./gradlew clean war` to build a traditional WAR file which will be created at `build/libs` directory.
```

Comment:
@ptuomola @vidakovic @vorburger Build is successful but WAR file is not generated in build/libs. 
> Task :war SKIPPED

Reply:
@avikganguly01 not sure what you mean, if I do `./gradlew war` at the root of the latest `develop` branch of the project, then I do get a `fineract-provider/build/libs/fineract-provider.war`. Are you perhaps by mistake just looking in `build/libs` instead of `fineract-provider/build/libs`? That changed, not because of FINERACT-1203 but as part of FINERACT-1171. Will you raise a PR to update the README to clarify this?

Reply:
Let's please discuss this further only on the comments in FINERACT-1203 instead of in 3 places.